demos/demo-stacked-area.js

```diff
@@ -26,12 +26,7 @@ function createStackedAreaChartWithTooltip(optionalColorSchema) {
         dataset;
 
     if (containerWidth) {
-        // dataset = aTestDataSet().withReportData().build();
-        // dataset = aTestDataSet().with3Sources().build();
         dataset = aTestDataSet().with6Sources().build();
-        // dataset = aTestDataSet().withLargeData().build();
-        // dataset = aTestDataSet().withGeneratedData().build();
-        // dataset = aTestDataSet().withSalesChannelData().build();
```

Comment:
I get why you deleted this, but I actually got value out of having these comments because that makes it easy to switch between different configurations of the demos for testing purposes.

Reply:
Yeah, it was useful to me too. It is easy to figure out though, just opening the databuilder!